src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10px 0 50px;
```

Comment:
```suggestion
.header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
  height: 60px;
```

Reply:
@volodymyr-soltys97   padding: 0 10px 0 50px; must be, with out it tests not gonna be passed